rasa/nlu/classifiers/embedding_intent_classifier.py

```diff
@@ -163,14 +163,14 @@ class EmbeddingIntentClassifier(EntityExtractor):
         # how often to calculate training accuracy
         EVAL_NUM_EPOCHS: 20,  # small values may hurt performance
         # how many examples to use for calculation of training accuracy
-        EVAL_NUM_EXAMPLES: 0,  # large values may hurt performance
+        EVAL_NUM_EXAMPLES: 10,  # large values may hurt performance
```

Comment:
let's keep default to `0`, since it excludes some data from training

Reply:
reverted the change in the last commit